op-node/p2p/cli/load_config.go

```diff
@@ -173,8 +173,14 @@ func loadDiscoveryOpts(conf *p2p.Config, ctx *cli.Context) error {
 	if err != nil {
 		return fmt.Errorf("failed to open discovery db: %w", err)
 	}
-
-	records := strings.Split(ctx.GlobalString(flags.Bootnodes.Name), ",")
+	records := p2p.OpBNBMainnetBootnodes
```

Comment:
The code's default value for records and the subsequent if statement are intertwined, causing confusion about their effects. I suggest modifying it as follows:
```suggestion
	var records []string
	if !ctx.IsSet(flags.Bootnodes.Name) {
		if rollupCfg.L2ChainID.Cmp(p2p.OpBNBTestnet) == 0 {
			records = p2p.OpBNBTestnetBootnodes
		} else {
		    records = p2p.OpBNBMainnetBootnodes
		}
	} else {
		records = strings.Split(ctx.GlobalString(flags.Bootnodes.Name), ",")
	}
```

Reply:
Good suggestion.